### ISSUE TITLE
Codesign failed. If you are using macOS 11+, Please download a new release from https://github.com/johnno1962/InjectionIII/releases

### ISSUE THREAD

**Issue by ShengfengLee** [johnno1962/InjectionIII#456]
Hello, when I try to perform a hot update, the console prints the following error.：

```
Compiler failed to build request

💉 ⚠️ Codesign failed. If you are using macOS 11+, Please download a new release from https://github.com/johnno1962/InjectionIII/releases
```

Injection version： 4.7.2
Xcode version：Version 14.3.1 (14E300b)
macOS version： 13.4.1 (c) (22F770820d)

**Comment by songxing10000** [comment]
same

**Comment by johnno1962** [comment]
Hi, sorry I havn't got back to you before on this. I'm traveling at the moment. How exactly are you using injection? You should just be able to quit the app and fall back to "standalone" injection and this problem won't occur for iOS.

**Comment by songxing10000** [comment]
It's strange that it's working normally today

**Comment by johnno1962** [comment]
That's not entirely strange. During the build, the code signing identity is saved for the project so when you retry it should work.

**Comment by ljw215** [comment]
same, how to solve

**Comment by johnno1962** [comment]
"Hi, sorry I haven't got back to you before on this; I'm traveling at the moment. How exactly are you using injection? You should just be able to quit the app and fall back to "standalone" injection and this problem won't occur for iOS in the simulator."

**Comment by johnno1962** [comment]
I'm back now and I was wondering what the cause of this could be and there were some changes to code signing in July. If you're still having this problem would it be possible to compare the 4.7.0 and 4.7.1 releases of the app and see if the problem is not present in the 4.7.0 version but present in 4.7.1? https://github.com/johnno1962/InjectionIII/releases

**Comment by johnno1962** [comment]
I've uploaded a new release candidate that does not attempt to code sign for the simulator which isn't necessary. If someone could give it a try please. https://github.com/johnno1962/InjectionIII/releases/tag/4.7.3

**Comment by johnno1962** [comment]
I found a problem with the code signing code in the finish and have also fixed that so this problem should be "fixed twice" if you want to upgrade to the newly released 4.7.3.
